packages/fiori/src/BarcodeScannerDialog.ts

```diff
@@ -68,10 +68,7 @@ type BarcodeScannerDialogScanErrorEventDetail = {
  * For a list of supported barcode formats, see the <ui5-link target="_blank" href="https://github.com/zxing-js/library">zxing-js/library</ui5-link> documentation.
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.fiori.BarcodeScannerDialog
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-barcode-scanner-dialog
+ * @extends UI5Element
  * @public
```

Comment:
- remove `@type` tag
- show/close method - remove `@method` and `@returns` tags and add explicit return type